cvat-ui/src/containers/annotation-page/standard-workspace/objects-side-bar/objects-list.tsx

```diff
@@ -589,6 +601,24 @@ class ObjectsListContainer extends React.PureComponent<Props, State> {
                     updateAnnotations([state]);
                 }
             },
+            MOVE_TO_PREVIOUS_LAYER: (event: KeyboardEvent | undefined) => {
+                preventDefault(event);
+                const state = activatedState(true);
+                if (state && !readonly && state.objectType !== ObjectType.TAG) {
+                    const newZOrder = state.zOrder - 1;
+                    state.zOrder = newZOrder;
+                    updateAnnotations([state]);
```

Comment:
It seems shortcut logic is different compared to logic that happens in menu clicks